src/loader.js

```diff
@@ -362,7 +362,7 @@ function setupModuleLoader(window) {
           component: function(name, options) {
             function factory($injector) {
               function makeInjectable(fn) {
-                if (angular.isFunction(fn)) {
+                if (isFunction(fn) || isArray(fn)) {
```

Comment:
if you just do `angular.isArray` you would break the loader as it is meant to be run without angular